torch/csrc/jit/python/script_init.cpp

```diff
@@ -1399,7 +1398,6 @@ void initJitScriptBindings(PyObject* module) {
             }
             return functions;
           })
-      .def("set_optimized", &CompilationUnit::set_optimized)
```

Comment:
Is this API still in use? How did you check this was ok to delete now?

Reply:
I removed it because CompilationUnit::set_optimized was deprecated.

Reply:
But that doesn't mean no end user is using it? Can we check that?

Reply:
No way to check. But the files have been marked as deprecated for a year.